dashboard/dashboard_script.py

```diff
@@ -0,0 +1,180 @@
+                with st.expander("Find out more"):
+                    Method_tab, Factors_tab = st.tabs(
+                        ["Methodology", "Intensity Factors"])
+                    with Method_tab:
+                        st.write("The demand and generation by fuel type (gas, coal, wind, nuclear, solar etc.) for each region is forecast several days ahead at 30-min temporal resolution using an ensemble of state-of-the-art supervised Machine Learning (ML) regression models. An advanced model ensembling technique is used to blend the ML models to generate a new optimised meta-model. The forecasts are updated every 30 mins using a nowcasting technique to adjust the forecasts a short period ahead. The carbon intensity factors are applied to each technology type for each import generation mix to calculate the forecast")
+                    with Factors_tab:
+                        st.table(self.intensity_factors_df())
+
+        with st.container():
+            st.header("Energy Generation by Fuel Type")
+            col1, col2 = st.columns([3, 1])
+            with col1:
+                st.altair_chart(self.generation_fuel_type(gen_df))
+            with col2:
+                with st.expander("Fuel Type Information"):
+                    st.write("""
+- **BIOMASS**: Organic materials used as a renewable energy source. It includes plant materials and animal waste that can be burned or converted to biofuels.
+- **CCGT (Combined Cycle Gas Turbine)**: A form of highly efficient energy generation that combines a gas-fired turbine with a steam turbine. 
+- **COAL**: Traditional fossil fuel used for generating electricity through combustion.
+- **NPSHYD (Non-Pumped Storage Hydro)**: Refers to the conventional hydroelectric generation where water flows through turbines to generate power, with no pumped storage components.
+- **NUCLEAR**: Power generated using nuclear reactions.Provides a stable base-load energy supply.
+- **OCGT (Open Cycle Gas Turbine)**: A type of gas power plant where air is compressed, mixed with fuel, and burned in a simple cycle. It's less efficient than CCGT but is often quicker to start and stop.
+- **OIL**: Oil-fired power plants burn petroleum or its derivatives to generate electricity. Similar to coal, it has large environmental impacts. 
+- **OTHER**: This category usually encompasses any generation sources not specifically listed, which could include experimental or less common technologies like tidal or certain types of bio-gas plants.
+- **PS (Pumped Storage)**: A type of hydropower generation where water is pumped to a higher elevation during low demand periods and released through turbines for electricity during high demand.
+- **WIND**: The use of wind turbines to convert wind energy into electricity. It's a clean, renewable source increasingly used worldwide, particularly in areas with strong, consistent winds.
+""")
+
+        with st.container():
```

Comment:
This code is visible on the dashboard. Has this been fixed with the latest push?